refs/schemas/priv/expected-behavior.md

```diff
@@ -0,0 +1,655 @@
+
+When Data Subject ID is provided, Data Subject is authenticated but is unknown by the System:
+- `OTHER` Demands: recommend human review and status = `UNDER-REVIEW`
+- For any other action : recommend status = `DENIED`, motive=`USER-UNKNOWN`
+
+When Data Subject ID is provided, the Data Subject is known by the System but failed to authenticate:
+- `OTHER` Demands: recommend human review and status = `UNDER-REVIEW`
+- `TRANSPARENCY.KNOWN` Demands: recommend status = `GRANTED`, and data = `NO`.
+- For any other action : recommend status = `DENIED`, motive=`IDENTITY-UNCONFIRMED`
+
+When Data Subject ID is provided, the Data Subject is known by the System and authenticated:
+- Regardless of restrictions
+    - `OTHER` Demands: recommend human review and status = `UNDER-REVIEW`
+    - `TRANSPARENCY.KNOWN` Demands: recommend status = `GRANTED`, and data = `YES`.
+    - `TRANSPARENCY.DATA-CATEGORIES` Demands: recommend status = `GRANTED`, and data = list of Data Categories that are included in any of the [Privacy Scope Triples](#privacy-scope-triples) included in the Eligible Privacy Scope.
+    - `TRANSPARENCY.PROVENANCE`, see [Resolving provenance in requests](#resolving-provenance-in-requests)
+    - `TRANSPARENCY.ORGANIZATION`, `TRANSPARENCY.POLICY`, `TRANSPARENCY.RETENTION`, `TRANSPARENCY.DPO`, `TRANSPARENCY.WHERE`, `TRANSPARENCY.WHO`, Demands: recommend status = `GRANTED`, and data = information corresponding to the request taken from configuration settings as defined under [Implications for Systems](./RFC-PRIV.md#design-implications-for-systems-implementing-PRIV) and under [Configuration and Prerequisites](#configuration-and-prerequisites).
+
+- With regards to Demand restrictions (if any) limiting the scope of the Demand:
+    - first, check for presence of incompatible restrictions (and if incompatible recommend status = `DENIED`, motive = `REQUEST-UNSUPPORTED`):
+        - [Consent Restriction](./RFC-PRIV.md#consent-restriction) with any other type of Restriction,
+        - [Consent Restriction](./RFC-PRIV.md#consent-restriction) within a Demand other than `REVOKE-CONSENT`,
+        - More than one [Capture Restriction](./RFC-PRIV.md#capture-restriction),
+        - More than one [Data Reference Restriction](./RFC-PRIV.md#data-reference-restriction)
+        - A Capture Restriction and any other restriction that is not a Privacy Scope Restriction, or a Data Reference Restriction,
+        - More than one [Date Range](./RFC-PRIV.md#date-range) Restriction.
+
+    - second, Calculate **Restriction Scope** and **Concerned Fragments**. This is done by processing every Restriction according to the following approach:
+        - at the beginning set **Restriction Scope** to be equal to the Eligible Privacy Scope of the Data Subject
+        **Concerned Fragments** is set to all Data Capture Fragments the `scope`s of which are included in the **Restriction Scope**.
```

Comment:
There is, further in the text, a note, explaining this:
 > Data Capture Fragments the scopes of which fall under the Restriction Scope are not the same as (but are a superset of) the Data Capture Fragments included in the Concerned Fragments scope.
 >
 > This is due to [Date Range](https://github.com/blindnet-io/product-management/blob/devkit-schemas/refs/schemas/priv/RFC-PRIV.md#date-range), [Capture Restriction](https://github.com/blindnet-io/product-management/blob/devkit-schemas/refs/schemas/priv/RFC-PRIV.md#capture-restriction) and [Provenance Restriction](https://github.com/blindnet-io/product-management/blob/devkit-schemas/refs/schemas/priv/RFC-PRIV.md#provenance-restriction), all of which might be used to select particular Data Capture Fragments out of many thay may be captured with the same Data Capture Fragment selector.

In fact the set of fragments concerned by Restriction scope != the set of Concerned Fragments

Reply:
The Restriction Scope is just another filter (like Date Range, Capture Restriction etc.) on the Capture Fragments.
Calculating the final Restriction Scope and applying it together with other filters at the end to the Fragments will save computing power (compared to filtering Fragments at every step).
Result will be the same.

Reply:
I am not sure I understand what you want to say.

Different restrictions might specify fragments that should be excluded from Concernes fragments, but can't be excluded from Restriction Scope. E.g. Data Range restriction.

Nothing stops you from caluclating the Restriction Scope frist, and the then calculating Concerned Fragments, as long as you take care of restrictions that specifically impact fragments (i.g. the set of fragments concerned by Restriction scope != the set of Concerned Fragments).

Reply:
My point is - we don't need a concept of `the set of fragments concerned by Restriction scope` since the Restriction scope is just a filter on the `scope` property, like Date Range is a filter on the `date` property.
When all the restrictions are applied, we end up with Concerned Fragments (which are, as you said, not equal to fragments filtered only by Restriction scope).

Anyway, this discussion belongs to FRD since it's an implementation detail and `expected-behavior` document should only specify which restriction targets which property and how the selection is done (related to value of the property).